src/move/emojicoin_dot_fun/sources/emojicoin_dot_fun.move

```diff
@@ -1169,6 +1214,36 @@ module emojicoin_dot_fun::emojicoin_dot_fun {
         );
     }
 
+    /// Since `init_module` will go over the transaction execution limit if we try to add the chat
+    /// emojis in the same transaction, we need to push the remaining chat emojis into the registry
+    /// post module publication.
+    /// We only add emojis that are not already in the registry by starting at the last index added
+    /// plus one, i.e., the current length of the chat emojis smart table.
+    public entry fun add_remaining_chat_emojis(
+        amount: u64,
+    ) acquires Registry, RegistryAddress {
+        let registry_ref_mut = borrow_registry_ref_mut();
+        let chat_emojis_ref_mut = &mut registry_ref_mut.chat_emojis;
+
+        let next_idx = smart_table::length(chat_emojis_ref_mut);
+        let chat_emojis = hex_codes::get_chat_emojis();
+        let num_chat_emojis = vector::length(&chat_emojis);
+        assert!(next_idx < num_chat_emojis, E_ALL_CHAT_EMOJIS_ADDED);
+
+        // AKA: let last_idx = min(next_idx + amount, num_chat_emojis);
+        let last_idx = if (next_idx + amount < num_chat_emojis) {
+            next_idx + amount
+        } else {
+            num_chat_emojis
+        };
+
+        // The next index to add is equal to the length of the smart table.
+        for (i in next_idx..last_idx) {
+            let emoji_bytes = *vector::borrow(&chat_emojis, i);
+            smart_table::add(chat_emojis_ref_mut, emoji_bytes, 0);
+        };
+    }
```

Comment:
I think it's not a problem for the first market registrant to pay I/O overhead of stuffing this into a table, rather than smart table

Makes things cleaner

Then we just need to stich into register market function, and all emojis will be initted by the time the first market is ready to go

Reply:
Done, only thing to note is that I just set the `fee` to 0 for now, although not sure we want to do that, but as noted in Slack there's no way to know the gas fees at runtime